src/ServiceControl.Monitoring/QueueLength/QueueLengthDataStore.cs

```diff
@@ -69,7 +69,7 @@ static MonitoredEndpointValues AggregateQueueLength(List<EndpointInstanceInterva
                 Points = intervals.SelectMany(t => t.Intervals)
                     .GroupBy(i => i.IntervalStart)
                     .OrderBy(g => g.Key)
-                    .Select(g => (double)g.Sum(i => i.TotalValue))
+                    .Select(g => g.Sum(i => i.TotalValue / returnOneIfZero(i.TotalMeasurements)))
```

Comment:
No :/. Please see this [comment](https://github.com/Particular/ServiceControl.Monitoring/pull/40#discussion_r129809797).